migrate-wordpress/README.md

```diff
@@ -149,13 +149,24 @@ Example (we chose to define the fact as environment variable):
       Create a machine to host the Wordpress server
 
 2. On Puppet Enterprise
-
-TODO(ody): Please fill this in.
+   - Wait for new GCE instance to come online after GCP infrastructure **Run #3**
+   - After checking that a fresh Wordpress is online, pin original source
+     instance to the `Migrate` node group in the `Classification` section of
```

Comment:
Are the classes described in the source? Is there a way to get a dump from the PE so we can show what categories gets which classes, and with which parameters.